.travis.yml

```diff
@@ -1,4 +1,8 @@
 language: node_js
 node_js:
-  - "0.8"
-  - "0.10"
\ No newline at end of file
+  - "node"
+  - "iojs"
```

Comment:
I have a preference for option 2. I prefer 2 only because `chalk` have updated supports_colors module. What this means is, if you are using the asciiffy CLI, you can `FORCE_COLOR` to bypass the default `tty` check. The use case maybe that a `jenkins` pipeline wants to print a banner in color and run some other shell commands on the slave, without the `FORCE_COLOR` env variable chalk will ignore any styles.